With Flexbox/css/index.css

```diff
@@ -0,0 +1,69 @@
+body {
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1 {
+    font-family: 'Bangers', cursive;
+}
+
+a {
+    color: #b0b0b0;
+    text-decoration: none;
+}
+
+ul.flex-container {
+    display: flex;
+    justify-content: space-around;
+}
+
+.main-container {
+    margin: 2.5% auto 0;
```

Comment:
You can remove the `margin: 2.5% auto 0` since we're adding flex rules to the parent container (`body`).

Also, keep the width at 800px.